tests/packages/install-subdir-exclude/meson.build

```diff
@@ -0,0 +1,45 @@
+# SPDX-FileCopyrightText: 2023 The meson-python developers
+#
+# SPDX-License-Identifier: MIT
+
+project('install-subdir-exclude', version: '1.0.0')
+
+py = import('python').find_installation()
+purelib = py.get_install_dir(pure: true)
+
+# We need two foo directories because of a bug in the Meson metadata generation.
+# https://github.com/mesonbuild/meson/issues/11539
+
+# Examples removed from https://mesonbuild.com/Reference-manual_functions.html#install_subdir
```

Comment:
"removed"? The examples are still in the linked Meson manual page. Maye "copied"? Although the example are not exactly the same, thus I would simply add a
```
# See https://mesonbuild.com/Reference-manual_functions.html#install_subdir
```
comment.

Reply:
This was poorly worded. I will update it.